src_c/joystick.c

```diff
@@ -671,16 +670,20 @@ MODINIT_DEFINE(joystick)
     }
     joy_instance_map = PyObject_GetAttrString(module, "_joy_instance_map");
     Py_DECREF(module);
```

Comment:
This is kinda strange, do you understand why it is there? I don't.

Reply:
It's for joystick events to have the old style ID attribute, IIRC (which is deprecated now)

Reply:
I mean the `Py_DECREF(module)`

(I thought Github would highlight my selection)

Reply:
A few lines above `module = PyImport_ImportModule("pygame.event");` returned a new reference to `module`, and after we have extracted the dict attribute from the event module, we have to decref it.

Perhaps you got confused thinking the joystick module object is being freed, which is not